components/Exercises/ExerciseCard/index.jsx

```diff
@@ -0,0 +1,55 @@
+import Heading from "@/components/Atoms/Text/Heading";
+import Image from "next/image";
+import HeadingLarge from "@/components/Atoms/Text/HeadingLarge";
+import HeadingTiny from "@/components/Atoms/Text/HeadingTiny";
+import styled from "styled-components";
+
+export default function ExerciseCard({ exercise }) {
+  return (
+    <>
+      <StyledExerciseCard>
+        <Image
+          src={exercise.imageUrl || "power.svg"}
+          alt="exercise image"
+          width={50}
+          height={50}
+        ></Image>
+        <article>
+          <HeadingLarge>{exercise.name}</HeadingLarge>
+          <Heading>Musclegroups</Heading>
+          <StyledList>
+            {exercise.muscleGroups.map((muscle) => (
+              <HeadingTiny>// {muscle}</HeadingTiny>
```

Comment:
```suggestion
              <HeadingTiny key={muscle}>// {muscle}</HeadingTiny>
```
Let's add a key here too